lib/src/main/res/layout/list_item_color.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Found the gist: https://gist.github.com/Mauker1/575e048fb2ee21856c35760babc35b3e